ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml

```diff
@@ -0,0 +1,37 @@
+base_images:
+  cli:
+    name: "4.14"
```

Comment:
The filename is `openshift-openshift-tests-private-master__rosacli.yaml`, by convention the file name should reflect the version under testing which I suppose is 4.14 for the `cli`  and `tests-private` in base_images are 4.14. Please rename to `openshift-openshift-tests-private-4.14__rosacli.yaml`

Reply:
@jhou1  I share some background here. This job is for the rosacli(rosa command line) testing, not aligned to OCP version, the job will build the rosacli with the latest code as always.  Do you think it is feasible？